README.md

```diff
@@ -8,6 +8,7 @@
 [![codecov](https://codecov.io/github/trixi-framework/TrixiParticles.jl/branch/main/graph/badge.svg?token=RDZXYbij0b)](https://codecov.io/github/trixi-framework/TrixiParticles.jl)
 [![SciML Code Style](https://img.shields.io/static/v1?label=code%20style&message=SciML&color=9558b2&labelColor=389826)](https://github.com/SciML/SciMLStyle)
 [![License: MIT](https://img.shields.io/badge/License-MIT-success.svg)](https://opensource.org/licenses/MIT)
+[![DOI](https://zenodo.org/badge/505448384.svg)](https://zenodo.org/doi/10.5281/zenodo.10797541)
```

Comment:
```suggestion
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10797541.svg)](https://zenodo.org/doi/10.5281/zenodo.10797541)
```
Add proper badge number

Reply:
Interesting. Why is this wrong? I just copied it from Zenodo:
<img width="872" alt="grafik" src="https://github.com/trixi-framework/TrixiParticles.jl/assets/44124897/10379c82-eeb3-453c-961d-09717905e04f">

Reply:
AFAIU, ...42 is the specific, current release, while ...41 is the super DOI that always points to the latest release. The badge link you used resolved to ..42 for me. I strongly suggest to always use the ...41 DOI whenever you reference TrixiP